### DISCUSSION TITLE
Yara application not working

### DISCUSSION THREAD

**Discussion by sujit211** [VirusTotal/yara#1969] [Q&A]
Hello,

I have noticed that the Yara tool is not functioning properly during startup. It used to initiate automatically without any problems, but now it fails to do so. This has been causing delays and inconveniences in my daily workflow.

I have attempted to troubleshoot the issue by checking for any recent updates or changes in configurations, but I have not been able to resolve it on my own. I believe it might be a technical glitch or a compatibility issue.

Also, during startup, I encountered an error message stating that the "vcruntime140.dll" file was missing. 

I would appreciate it if you could look into this matter and provide assistance in resolving the problem as soon as possible. 
Thank you!

**Comment by plusvic** [comment]
Can you provide more information about what you are doing? For example, I would need to know if you are using the command-line version of YARA, which command are you trying, and what's exactly the error you are getting.

**Comment by sujit211** [comment]
Hello Sir,

I started by downloading the 64-bit Yara utility from the Virustotal website, and I then extracted the Yara64 and Yarac64 programmes. And a tiny window appeared and vanished quickly whenever I attempted to open one of these two. Additionally, it keeps happening. Also, I checked my antivirus software to see if it was attempting to prevent it, but that was not the case. I also tried opening the yara64 application in a sandbox, wherein I got an error stating "vcruntime140.dll" file was missing. 

I'm new to this tool, after all. I would be grateful for your assistance with this. 

Thank you!
